platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/model/WiringModel.java

```diff
@@ -136,6 +136,23 @@ String generateWiringDiagram(
     @NonNull
     OutputWire<Instant> buildHeartbeatWire(final double frequency);
 
+    /**
+     * Lower a metaphorical anchor into the JVM that prevents the JVM from shutting down. This is useful if there are no
+     * other non-daemon threads running in the JVM. Before the JVM is permitted to exit, the anchor must be raised via
+     * {@link #raiseJvmAnchor()}.
+     * <p>
+     * Calling this method when the anchor is already lowered has no effect.
+     */
+    void lowerJvmAnchor();
```

Comment:
Anchor lowering stops the motion of that which lowered the anchor.    I don't know that lowering an anchor is the right metaphor.  Inserting wrench into gears is a better metaphor.  What about `insertWrenchIntoJVMExitGears`?   Too cumbersome.   Perhaps a straightforward purposeful method name like `preventJVMExit()` and `allowJVMExit()`.

Reply:
I've renamed these methods as you have suggested.

I did not rename the helper class JvmAnchor... not exactly sure what to call it. Since this is more of an internal detail, I think it's less important to rename. But if you have any good suggestions, I'm still open to the idea.